actions/global_actions.jsx

```diff
@@ -437,7 +441,13 @@ export function emitUserLoggedOutEvent(redirectTo = '/', shouldSignalLogout = tr
                 BrowserStore.signalLogout();
             }
 
-            clientLogout(redirectTo);
+            BrowserStore.clear();
```

Comment:
No need to specify exclude keys here: the keys listed previously weren't managed by the browser store and lived on anyway. `selected_teams`, in particular, seems to have been entirely unused.

Reply:
My only concern is the RECENT_EMOJI_KEY.  So on logout then login, the recently used emojis are still there?

Reply:
Correct: this threw me off for a while, believing I could influence what was preserved after logout, but it turns out `BrowserStore` doesn't manage keys outside of the `redux-persist` stuff. Recently used emojis -- alongside my new bits in the localStorage -- are preserved on logout.